src/apps/answers/service.py

```diff
@@ -366,7 +367,23 @@ async def validate_multiinformant_assessment(
             activity, flow = await asyncio.gather(activity_future, flow_future)
             if not activity and not flow:
                 raise MultiinformantAssessmentInvalidActivityOrFlow()
+            
+
+        is_admin = await AppletAccessCRUD(self.session).has_any_roles_for_applet(
+            respondent_subject.applet_id,
+            respondent_subject.user_id,
+            Role.managers(),
```

Comment:
We need this to be only owners and managers

```suggestion
            [Role.OWNER, Role.MANAGER],
```